src/swap/saga.ts

```diff
@@ -167,12 +165,6 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
       throw new Error('Unknown token network')
     }
 
-    const wallet = yield* call(getViemWallet, networkConfig.viemChain[network])
```

Comment:
`getViemWallet` calls `getLockableViemWallet` which throws if it cannot find the keychain account. it's also called inside `sendPreparedTransaction` which is called later in this saga, so i thought it safe to remove (rather than try to prevent the throw here)